.github/workflows/releasenotes-gen.yml

```diff
@@ -1,6 +1,7 @@
 name: "Generate release notes"
 
 on:
+  workflow_dispatch: null
```

Comment:
Please specify branch here, like:
```
workflow_dispatch:
  branches: [ master ]

```
If we only run action on push to master, we should only dispatch from master.

Reply:
done

Reply:
Same as https://github.com/checkstyle/checkstyle/pull/12580#discussion_r1061487465